### PR TITLE
Add Notebook 02: Variables, operators, & types

### LINKED ISSUE #3: Slides: Variables, Operators, & Types
Draw from:
- Intro course: [*Fundamentals*](https://github.com/uc-python/intro-python-datasci/blob/master/notebooks/02-Fundamentals.ipynb)
- Bana 6043: [*Lesson 1b*](https://github.com/bradleyboehmke/uc-bana-6043/blob/main/book/01-module/lesson-1b.ipynb)

### PR DESCRIPTION
**Pull request by eswan18**
This pulls mostly from material in the Fundamentals notebook of the training class. However, I removed a fair amount of Pandas details and – with all the extra space – I added an overview of some types we didn't usually cover: tuples, sets, bytes, and niche numerics.

I still think this section is a little bit short, though everything always longer than I expect. I'm open to coming back and adding more content to this if anyone thinks there's a topic that belongs here.

### CODE REVIEW

File: notebooks/02-Variables-Operators-Types.ipynb [hunk 1]
```diff
@@ -0,0 +1,1744 @@
...
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "<font class=\"your_turn\">\n",
+    "    Your Turn\n",
+    "</font>\n",
```
Comment (augustopher) [inline notebooks/02-Variables-Operators-Types.ipynb]: Should probably convert to a header.

Comment (eswan18) [reply notebooks/02-Variables-Operators-Types.ipynb]: Good point, updating.

Comment (eswan18) [reply notebooks/02-Variables-Operators-Types.ipynb]: Fixed in dbb1bf6

File: notebooks/02-Variables-Operators-Types.ipynb [hunk 2]
```diff
@@ -0,0 +1,1741 @@
...
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Your Turn\n",
```
Comment (augustopher) [inline notebooks/02-Variables-Operators-Types.ipynb]: Up to you: most of these "Your Turn" headers in other notebooks have been `h3`, with `h2` as a section divider. Not a huge deal though.

**Review by augustopher — APPROVED** [review]

File: notebooks/02-Variables-Operators-Types.ipynb (hunk 2 above)
Comment (eswan18) [reply notebooks/02-Variables-Operators-Types.ipynb]: They do say that inconsistency is the hobgoblin of sloppy teachers. I'll make a quick fix.

Comment (augustopher) [reply notebooks/02-Variables-Operators-Types.ipynb]: > They do say that inconsistency is the hobgoblin of sloppy teachers.

Can this be added to a style guide somewhere?

Comment (eswan18) [reply notebooks/02-Variables-Operators-Types.ipynb]: I am about 95% sure that it's actually in the original Code of Hammurabi, I should have consulted that before I dove into making these slides.

Comment (eswan18) [reply notebooks/02-Variables-Operators-Types.ipynb]: Fixed in [a706749](https://github.com/uc-python/python-training-cincinnati-insurance/pull/33/commits/a7067491459c1ba9392970d8ebe24c1a35669a1b)

**Review by augustopher — APPROVED** [review]

**Pull request merged**